api/types/kustomization.go

```diff
@@ -188,6 +188,7 @@ const (
 	deprecatedPatchesJson6902Message           = "# Warning: 'patchesJson6902' is deprecated. Please use 'patches' instead." + " " + deprecatedWarningToRunEditFix
 	deprecatedPatchesStrategicMergeMessage     = "# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead." + " " + deprecatedWarningToRunEditFix
 	deprecatedVarsMessage                      = "# Warning: 'vars' is deprecated. Please use 'replacements' instead." + " " + deprecatedWarningToRunEditFixExperimential
+	deprecatedCommonLabelsWarningMessage       = "# Warning: 'commonLabels' is deprecated. Please use 'labels' instead."
```

Comment:
question: is this something that `kustomize edit fix` also fixes automatically? If yes, should we append the message from line 184 here as well?

Reply:
Great catch! I didn't know about this feature. I tested `kustomize edit fix` and verified that fix does replace `commonLabels` with `labels`. Here is relevant [fix code](https://github.com/kubernetes-sigs/kustomize/blob/master/kustomize/commands/edit/fix/fix.go#L71).

Per your suggestion , I appended the `deprecatedWarningToRunEditFix` message.

Reply:
Amazing, thanks for updating it!